hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/SolvencyPreCheckTest.java

```diff
@@ -178,8 +179,10 @@ void testCheckSolvencyWithIllegalParameters() {
             final var payer = ALICE.account().copyBuilder().tinybarBalance(FEE).build();
 
             // then
-            assertThatThrownBy(() -> subject.checkSolvency(null, payer, FEE)).isInstanceOf(NullPointerException.class);
-            assertThatThrownBy(() -> subject.checkSolvency(txInfo, null, FEE)).isInstanceOf(NullPointerException.class);
+            assertThatThrownBy(() -> subject.checkSolvency(null, payer, new Fees(FEE, 0, 0)))
```

Comment:
Probably makes sense to turn `FEE` into a `Fees` instance. Do we use the primitive value at all?

Reply:
Replaced with `Fees` instance.